plugins/custom-cols.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: custom-cols
+spec:
+  version: "v0.0.2"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/webofmars/kubectl-custom-cols/archive/v0.0.2.tar.gz
+    sha256: "f01d1aa63c320989071baa88d7b0a8fefa1147e0da8b9edf32874def21c84736"
+    files:
+    - from: "./kubectl-custom-cols-0.0.2/kubectl-custom-cols"
```

Comment:
FYI instead of `./kubectl-custom-cols-0.0.2/kubectl-custom-cols` you can do `./kubectl-custom-cols-*/kubectl-custom-cols` as well.

Reply:
done